androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/LauncherActivityMetadata.java

```diff
@@ -35,6 +36,8 @@
  */
 public class LauncherActivityMetadata {
 
+    private static final String TAG = "LauncherActivityMetadata";
```

Comment:
We don't use this anywhere (nor the `import`).

Reply:
I will remove this later. I was trying to test the result there.